data-upload/example_synthetic.jsonld

```diff
@@ -1,10 +1,11 @@
 {
   "@context": {
-    "nb": "http://neurobagel.org/vocab/",
-    "snomed": "http://purl.bioontology.org/ontology/SNOMEDCT/",
-    "nidm": "http://purl.org/nidash/nidm#",
     "cogatlas": "https://www.cognitiveatlas.org/task/id/",
+    "nb": "http://neurobagel.org/vocab/",
     "ncit": "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#",
+    "nidm": "http://purl.org/nidash/nidm#",
+    "snomed": "http://purl.bioontology.org/ontology/SNOMEDCT/",
+    "np": "https://github.com/nipoppy/pipeline-catalog/tree/main/processing",
```

Comment:
why the change in order? just the output of generate_context()?

Reply:
Also: this jsonld doesn't actually have any pipeline instances, right? Why is `np` then in the context? Do we just slap the whole context on no matter if we need it?

Reply:
> why the change in order? just the output of generate_context()?

Yes, it's just the (arbitrary) order in which the namespaces appear in the list `ALL_NAMESPACES`:
```bash
ALL_NAMESPACES = [COGATLAS, NB, NCIT, NIDM, SNOMED, NP]
```

> Also: this jsonld doesn't actually have any pipeline instances, right? Why is np then in the context? Do we just slap the whole context on no matter if we need it?

That's the idea, and that's what we've been doing so far. We only have one `generate_context()`, and it includes everything we model, grabbed from `models.py`. If we wanted the `@context` to be jsonld-specific (i.e., different for the output of every CLI command), the logic of `generate_context()` would need to be a lot more complex.

Anyways, since most subject attributes are optional, even if we include only e.g., phenotypic-related terms/namespaces in the `@context` for a pheno-only JSONLD, it does not guarantee those terms will appear in every JSONLD generated by `bagel pheno` - rather, it would depend on the dataset. To avoid these discrepencies, I think it still makes sense to have all namespaces/terms we support in the `@context`.

This also makes it much easier to test if the context we generate captures everything needed, since we expect it to be static.